nncf/quantization/algorithms/fast_bias_correction/onnx_backend.py

```diff
@@ -79,25 +80,24 @@ def get_sub_input_output_names(subgraph: onnx.ModelProto) -> Tuple[str, str]:
     def create_input_data(
         shape: Tuple[int], data: List[np.ndarray], input_name: str, channel_axis: int
```

Comment:
Please adjust type hints here and below.

Reply:
done